### PR TITLE
[#IP-197] Add pipelines for `@pagopa/io-functions-eucovidcerts`

### PR DESCRIPTION
**Pull request by gquadrati**
Code review and deploy pipelines for `@pagopa/io-functions-eucovidcert`

### CODE REVIEW

**Review by balanza — CHANGES_REQUESTED** [review]
I would not use `io-backend-projects` as project, as it's for core backend services.

Create a new one, maybe `eucertcovid-projects`

**Comment by gquadrati** [comment]
> I would not use `io-backend-projects` as project, as it's for core backend services.
> 
> Create a new one, maybe `eucertcovid-projects`

Done in [e6a88af](https://github.com/pagopa/gitops/pull/68/commits/e6a88af935daf7c31fb6944c3c1aec06983e0fbd)

File: azure-devops/projects/eucertcovid-projects/project.tf [hunk 1]
```diff
@@ -0,0 +1,22 @@
+locals {
+  azure_devops_org = "pagopa-io"
+}
+
+resource "azuredevops_project" "project" {
+  name               = "eucertcovid-projects"
```
[the remark this answers is not in the document]
Comment (pasqualedevita) [reply azure-devops/projects/eucertcovid-projects/project.tf:6]: I'm going a refactor, stay tuned ;)

[the remark this answers is not in the document]
Comment (gquadrati) [reply azure-devops/projects/eucertcovid-projects/project.tf]: Sorry, I just copied what @balanza asked in previous comment, I didn't realize the inversion

[the remark this answers is not in the document]
Comment (pasqualedevita) [reply azure-devops/projects/eucertcovid-projects/project.tf]: wait 10 minutes to review.

[the remark this answers is not in the document]
Comment (balanza) [reply azure-devops/projects/eucertcovid-projects/project.tf]: Sorry, my bad

[the remark this answers is not in the document]
Comment (gquadrati) [reply azure-devops/projects/eucertcovid-projects/project.tf]: Mine too, I should have check :)

**Comment by pasqualedevita** [comment]
@balanza @gquadrati please review now pipeline definition is more compact with less repeat 
https://github.com/pagopa/gitops/pull/68/commits/d855032f9e73a6b2c2dd9e21aed17bfbfb06f4f6

**Review by pasqualedevita — APPROVED** [review]

**Comment by pasqualedevita** [comment]
@balanza @gquadrati manca il pagopa-bot sul repo

**Review by pasqualedevita — CHANGES_REQUESTED** [review]
permissions

**Comment by gquadrati** [comment]
> permissions

Right, we need to fix all permissions in new function...

**Review by pasqualedevita — APPROVED** [review]

**Comment by balanza** [comment]
> @balanza @gquadrati please review now pipeline definition is more compact with less repeat
> [d855032](https://github.com/pagopa/gitops/commit/d855032f9e73a6b2c2dd9e21aed17bfbfb06f4f6)

Very beautiful I like

**Pull request merged**